src/config.rs

```diff
@@ -0,0 +1,275 @@
+use serde::{Deserialize, Serialize};
+use std::error;
+use std::io::Write;
+use uuid::Uuid;
+
+use tiny_http::{Response, Server};
+
+use std::collections::HashMap;
+use url::Url;
+
+use serde_json::Value;
+use tungstenite::{connect, Message};
+
+use ha_api::types::{GetAccessTokenResponse, RegisterDeviceResponse};
+use ha_api::HomeAssistantAPI;
+
+// HA creates tokens for 10 years so we do the same
+const LONG_LIVED_TOKEN_VALID_FOR: u32 = 365;
```

Comment:
This looks a lot like 1 year?

Reply:
The default in HA when you manually create one is 10 years. But open to setting this to what we would like.

Reply:
My main point is around the comment not matching the value. I don't have a strong preference around how long the token is.

Assuming generating a new one isn't a PITA, a year is probably fine.

Reply:
ah yeah I originally had 10 years, but the API freaked out when I passed it that. I'll update the comment.